packages/workshop-app/app/components/edit-readme.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { LaunchEditor } from '~/routes/launch-editor.tsx'
+
+export function EditReadme({
+	name,
+	stepPath,
+}: {
+	name: string
+	stepPath: string
+}) {
+	const [altDown, setAltDown] = React.useState(false)
+
+	React.useEffect(() => {
+		if (ENV.KCDSHOP_DEPLOYED) return
+		const set = (e: KeyboardEvent) => setAltDown(e.altKey)
+		document.addEventListener('keydown', set)
+		document.addEventListener('keyup', set)
+		return () => {
+			document.removeEventListener('keyup', set)
+			document.removeEventListener('keydown', set)
+		}
+	}, [])
+
+	// prevent Alt+click from trying to save the file when KCDSHOP_DEPLOYED is true
+	// by using button instead of <a> tag or <Link>
+	const handleClick = () => {
+		const githubPath = ENV.KCDSHOP_GITHUB_ROOT.replace(/tree|blob/, 'edit')
```

Comment:
```suggestion
		const githubPath = ENV.KCDSHOP_GITHUB_ROOT.replace(/\/tree\/|\/blob\//, '/edit/')
```

This will make it less likely to replace the wrong thing.

Reply:
OK